verification/src/http_server/handlers/verification/solidity/types.rs

```diff
@@ -57,6 +55,17 @@ impl TryFrom<FlattenedSource> for CompilerInput {
     }
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct StandardJson {
```

Comment:
maybe add this struct as input for [handler](https://github.com/blockscout/blockscout-rs/blob/2cb273702cab077a5ff8a9bfe07e87d9c9f4c58d/verification/src/http_server/handlers/verification/solidity/standard_json.rs#L17) function?